src/Tribe/Attendees.php

```diff
@@ -486,7 +487,7 @@ public function screen_setup() {
 		} else {
 			$this->attendees_table = new Tribe__Tickets__Attendees_Table();
 
-			$this->maybe_generate_csv();
+			$this->maybe_generate_csv( 'all' === ( $_GET['event_id'] ?? '' ) ); // phpcs:ignore WordPress.Security
```

Comment:
This is too broad - it should be for something specific (like`WordPress.Security.NonceVerification.Recommended` verification, which should happen earlier)

Reply:
Also recall you can use tribe_get_request_var to bypass the nonce warning - where it's not appropriate (I don't think there is a nonce available here)